lib/api/error-handler.ts

```
@@ -0,0 +1,113 @@
+      },
+    },
+    { status }
+  );
+}
+
+/**
+ * Handles API route errors with appropriate logging and response formatting
+ */
+export function handleApiError(
+  error: unknown,
+  context: string = 'API'
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)